### ISSUE TITLE
epochs to train:200 vs.1000

### ISSUE THREAD

**Issue by Shuolongbj** [tkipf/gae#3]
Epoch: 0198 train_loss= 0.40438 train_acc= 0.80021 val_roc= 0.91772 val_ap= 0.93526 time= 0.42112
Epoch: 0199 train_loss= 0.40435 train_acc= 0.80037 val_roc= 0.91785 val_ap= 0.93544 time= 0.51987
Epoch: 0200 train_loss= 0.40434 train_acc= 0.80052 val_roc= 0.91780 val_ap= 0.93540 time= 0.49462
Optimization Finished!
Test ROC score: 0.923083293426
Test AP score: 0.929091873277

Epoch: 0999 train_loss= 0.39955 train_acc= 0.89461 val_roc= 0.86105 val_ap= 0.85885 time= 0.43444
Epoch: 1000 train_loss= 0.39954 train_acc= 0.89462 val_roc= 0.86104 val_ap= 0.85884 time= 0.45543
Optimization Finished!
Test ROC score: 0.873462980099
Test AP score: 0.893914158688

**Comment by tkipf** [comment]
There's some overfitting happening. Early stopping helps, but you can also play around with other types of reglarization (e.g. Dropout)
